scripts/ros-generate-cache.sh

```diff
@@ -18,7 +18,7 @@
 # Copyright (c) 2019-2020 LG Electronics, Inc.
 
 readonly SCRIPT_NAME="ros-generate-cache"
-readonly SCRIPT_VERSION="1.5.3"
+readonly SCRIPT_VERSION="1.6.0"
```

Comment:
Commit body:

> leave ROS_VERSION and ROS_PYTHON_VERSION setting for
>  "conditional" attributes on superflore itself

I don't understand what this means.

Reply:
I meant that we'll leave it on this superflore implementation:
https://github.com/ros-infrastructure/superflore/pull/280/commits/5b9014b14c50e6a79c6bce4e5c8de28b4bb270b4
to set ROS_VERSION and ROS_PYTHON_VERSION based on the distribution name

Reply:
In that case:

`don't set ROS_PYTHON_VERSION (v1.6.0)` -> `don't export ROS_VERSION &  ROS_PYTHON_VERSION (v1.6.0)`

`leave ROS_VERSION and ROS_PYTHON_VERSION setting for "conditional" attributes on superflore itself` -> `superflore no longer reads ROS_VERSION and ROS_PYTHON_VERSION from its environment but derives them from the specified ROS_DISTRO, so don't export them.